frontend/frontend.go

```diff
@@ -181,8 +217,21 @@ func (f *Frontend) ArmResourceListByResourceGroup(writer http.ResponseWriter, re
 
 func (f *Frontend) ArmResourceRead(writer http.ResponseWriter, request *http.Request) {
 	ctx := request.Context()
-	logger := ctx.Value(ContextKeyLogger).(*slog.Logger)
-	versionedInterface := ctx.Value(ContextKeyVersion).(api.Version)
+	logger, ok := LoggerFromContext(ctx)
+	if !ok {
+		logger = DefaultLogger()
```

Comment:
I am also thinking, if the default logger is enough, why do we pass it in the context?